GG/src/Font.cpp

```diff
@@ -2242,7 +2242,7 @@ void Font::HandleTag(const std::shared_ptr<FormattingTag>& tag, double* orig_col
                     } else {
                         well_formed_tag = false;
                     }
-                } catch (boost::bad_lexical_cast) {
+                } catch (const boost::bad_lexical_cast&) {
```

Comment:
could make these just `catch (...)`

Reply:
It should only catch the `boost::bad_lexical_cast` emitted by the lexical_cast, not every exception.